lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1814,7 +1814,7 @@ namespace Js
             // Free unused bytes
             Assert(cbNeeded + 1 <= cbUtf8Buffer);
             *ppSourceInfo = Utf8SourceInfo::New(this, utf8Script, (int)length,
-                cbNeeded, pSrcInfo, isLibraryCode, scriptSource);
+                cbNeeded, pSrcInfo, isLibraryCode, nullptr);
```

Comment:
consider commenting on the parameter name (nullptr /* scriptSourceVar */) - alternatively, isn't that parameter default? If so, maybe just remove it altogether?